src/components/OurValues/OurValues.js

```diff
@@ -21,12 +21,12 @@ export default function OurValues() {
     )
   })
   return (
-    <CommunityValuesSection>
+    <Section>
       <Headline1>{titleData.headline1}</Headline1>
```

Comment:
Sometimes above the return you'll see the properties off the object get destructured then inserted into the JSX.

Like const { headline1, headline2 } = titleData;

then <Headline1> {headline1} <Headline2>

Depending on the linter setup you'll actually get yelled at for "prefer object destructure". Another practise to keep in mind (and to use on any tech tests that may come your way:))